features/unset-value.yml

```
@@ -0,0 +1,6 @@
+name: unset
+description: The `unset` keyword can be applied to any CSS property, removing any values previously declared by the author, user, or browser. For inherited properties (like `color`) the property will inherit the parent value, while non-inherited properties (like `display`) will default to the specification-defined initial value for that property (`inline`). In order to fallback to the browser default styles instead, authors can use `revert` or `revert-layer`.
```

This will exceed the length limit (300 characters). Here's my take on shortening this:

```suggestion
description: The `unset` keyword resets any CSS property, removing any values previously set by the author, user, or browser. Inherited properties (like `color`) reset to the parent element's computed value. Non-inherited properties (like `display`) reset to the specified initial value (`inline`).
```